### ISSUE TITLE
Cassandra Error after restart

### ISSUE THREAD

**Issue by Gingonic** [Mashape/kong#468]
Hi there,

After restarting Kong, (container based, started over marathon) I have got the cryptic:

```
[INFO] Connecting to the database...
[ERR] Cassandra error: Cassandra returned error (Invalid): "Invalid column name path because it conflicts with an existing column"
```

I would not really enjoy having reconfuigure the entire proxy. An idea where this might come from?

Thanks in advance.

**Comment by Gingonic** [comment]
Short edit, version might help

```
[INFO] Using configuration: /etc/kong/kong.yml
[INFO] Kong version.......0.3.2
       Proxy HTTP port....8000
       Proxy HTTPS port...8443
       Admin API port.....8001
       dnsmasq port.......8053
       Database...........cassandra keepalive=60000 hosts=services.mycompany.io port=9042 timeout=1000 keyspace=kong
[INFO] Connecting to the database...
[ERR] Cassandra error: Cassandra returned error (Invalid): "Invalid column name path because it conflicts with an existing column"
```

**Comment by Gingonic** [comment]
And bumping up to 0.4.1 did not help

```
[INFO] Using configuration: /etc/kong/kong.yml
[INFO] Kong version.......0.4.1
       Proxy HTTP port....8000
       Proxy HTTPS port...8443
       Admin API port.....8001
       dnsmasq port.......8053
       Database...........cassandra keepalive=60000 hosts=services.mobilecity.io port=9042 timeout=1000 keyspace=kong
[INFO] Connecting to the database...
[ERR] Cassandra error: Cassandra returned error (Invalid): "Invalid column name path because it conflicts with an existing column"
```

**Comment by thibaultCha** [comment]
Hi,

Can you connect to your Cassandra cluster with cqlsh and execute:

```sql
USE <your_keyspace>;
DESCRIBE TABLES;
```

Especially look for a table named `schema_migrations`. In case it is here, please also share the output of:

```sql
SELECT * FROM schema_migrations;
```

That would be really helpful and that way I could give you instructions on how to fix your cluster without re-configuring everything.

Could you also tell us how many Cassandra nodes are you running ? Thank you.

**Comment by Gingonic** [comment]
Hi,

I have saved every configuration request, so I could rebuild everything with not too much pain.

I've saved the broken Cassandra state though, so if you expect to find some interesting bug that would improve Kong itself I would search were it went wrong but if not I would let this one go if I'm the first one to experience it. 

As we are still in staging I'have only one instance running.

Let me know if this might be of general interess and thanks for the response.
